Lesson4/Presentation/Account/AccountViewController.swift

```diff
@@ -0,0 +1,129 @@
+    }
+       
+    //MARK: - Helpers
+    private func configure() {
+        postsTableView.delegate = self
+        postsTableView.dataSource = self
+        postsTableView.estimatedRowHeight = 400
+        configureViews()
+    }
+    
+    private func obtainPostsData() {
+        DispatchQueue.global(qos: .background).async { [weak self] in
+            guard let strongSelf = self else { return }
+            strongSelf.dataManager.getPosts { data in
+                DispatchQueue.main.async {
+                    strongSelf.posts = data
+                    strongSelf.postsTableView.reloadData()
+                }
+            }
+        }
+    }
+    
+    private func obtainUserData() {
+        let tabBar = tabBarController as! MainTabBarViewController
+        user = tabBar.user
+        configureProfile()
+    }
+    
+    // Set up elements
+    private func configureViews() {
+        moreInfoButton.layer.cornerRadius = moreInfoButton.frame.height / 2
+        moreInfoButton.backgroundColor = .gray
+        changeStatusButton.layer.cornerRadius = changeUserStatusButton.frame.height / 2
+        changeStatusButton.backgroundColor = .gray
+        addNewPostButton.layer.cornerRadius = addNewPostButton.frame.height / 2
+        addNewPostButton.backgroundColor = .gray
+    }
+    
+    private func configureProfile() {
+        guard let user = user else { return }
+        userNameTextLabel.text = user.name
+        userAvatarImageView.image = user.avatar
+        userStatusTextLabel.text = user.status
+        userTownTextLabel.text = user.city
+        userPhoneNumberTextLabel.text = user.phone
+    }
+    
+    // Segue
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == Constants.detailedPostSeque.rawValue,
+               let viewController = segue.destination as? DetailedPostViewController,
+                let post = sender as? Post {
+                    viewController.post = post
+        }
+        if segue.identifier == Constants.editingAccountSeque.rawValue, let viewController = segue.destination as? EditingAccountViewController, let user = sender as? User {
+            viewController.user = user
+            viewController.editingAccountDelegate = self
+        }
+    }
+}
+
+// MARK: - Table view data source
+extension AccountViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return posts.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as? NewsTableViewCell else { return UITableViewCell() }
+        cell.configure(posts[indexPath.row])
+        return cell
+    }
+}
+
+//MARK: - Edit account delegate
```

Comment:
```suggestion
// MARK: - EditingAccountDelegate
```
чаще всего пишут просто так, чтобы контекст файла понимать